02-technical-guides/virtual-hierarchies/main.ipynb

```diff
@@ -11,53 +11,80 @@
     "tags": []
```

Comment:
<p>Took out...</p><p> </p><p>"The trade-off between a smaller memory footprint and faster commit times versus offering a slightly reduced set of features is definitely worth it."</p><p> </p><p>as this was very opinionated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>